xdr2/bench_test.go

```diff
@@ -14,14 +14,12 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-package xdr_test
+package xdr
```

Comment:
I am sure @davecgh has a reason to split test from package.  Seems like a gratuitous change.